Makefile

```diff
@@ -145,7 +145,9 @@ lint-ui:  ## Lint the UI code
 
 build-ui:  ## Builds the UI
 	cd $(MANAGER_UI_DIR) && npm run build
-
+	mkdir -p ./bin
+	${OPTS} GOBIN=${PWD}/bin go get github.com/rakyll/statik
+	./bin/statik -src=$(MANAGER_UI_DIR)/dist -dest ./cmd/hypervisor -f
```

Comment:
Is it possible to change it to the same as above (`${PWD}/bin`) or to change above to the same as this line (`GOBIN=./bin`)?

Reply:
`GOBIN=./bin` is not allowed. `golang` only accepts absolute paths for `GOBIN` FYI.
I'll do it the other way.

Reply:
@evanlinjin You're right, I forgot about that. Then, I'd change all `./bin`s to `${PWD}/bin`s then for consistency and readability.